TeamCode/src/main/java/org/firstinspires/ftc/teamcode/opmodes/redCloseAuto.java

```diff
@@ -0,0 +1,113 @@
+package org.firstinspires.ftc.teamcode.opmodes;
+
+import com.qualcomm.robotcore.eventloop.opmode.Autonomous;
+import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode;
+
+import org.firstinspires.ftc.teamcode.hardware.Drivebase;
+import org.firstinspires.ftc.teamcode.hardware.VisionCamera;
+import org.firstinspires.ftc.teamcode.utility.CubeSide;
+import org.firstinspires.ftc.teamcode.utility.FieldSide;
+
+//Names the autonomous on the control hub dropdown.
+@Autonomous(name = "pixelMarksAuto")
+public class redCloseAuto extends LinearOpMode {
+    @Override
+    //This method is needed for teleOp and auto to run at all.
+    public void runOpMode() {
+        //Runs after the robot has been initialized.
+        //Assign vars here and create instances of needed classes.
+
+        telemetry.setAutoClear(false);
```

Comment:
If we always set telemetry autoclear to true, why is it false here? If this is an exception, you should add a comment on why.